src/modules/auth/clients/FormFields.js

```diff
@@ -7,9 +7,16 @@ import { useTranslation } from 'react-i18next';
 import { Controller } from "react-hook-form";
 
 // The usual text input
-export function TextInput ({ name, register, labelName, disabled }) {
+export function TextInput ({ name, register, errors, labelName, disabled }) {
 	const { t } = useTranslation();
-	const reg = register(name);
+	const reg = register(
+		name,
+		(name == "preferred_client_id") && {
+			validate: {
+				validation: value => (/^[-_a-zA-Z0-9]{8,64}$|^$/).test(value) || t("ClientFormField|Invalid format, should have minimum of 8 characters"),
```

Comment:
@Aringoaway As per my comment in locales `Invalid format, input should have minimum of 8 characters`